modules/ephemeral-storage-csi-inline-overview.adoc

```diff
@@ -15,7 +15,7 @@ This feature allows you to specify CSI volumes directly in the `Pod` specificati
 By default, {product-title} supports CSI inline ephemeral volumes with these limitations:
 
 * Support is only available for CSI drivers. In-tree and FlexVolumes are not supported.
-* The Shared Resource CSI Driver supports inline ephemeral volumes as a Technology Preview feature.
+* The Shared Resource CSI Driver supports inline ephemeral volumes as a Technology Preview feature, but only for the purpose of accessing `Secrets` or `ConfigMaps` across multiple namespaces.
```

Comment:
```suggestion
* The Shared Resource CSI Driver supports using inline ephemeral volumes to access `Secrets` or `ConfigMaps` across multiple namespaces as a Technology Preview feature.
```

Reply:
@rolfedh can we say `... inline ephemeral volumes only to access ...`

i.e. add `only`

Reply:
> @rolfedh can we say `... inline ephemeral volumes only to access ...`
> 
> i.e. add `only`

Yes. that works.